app/controllers/pre_application/project_enquiry/what_project_does_controller.rb

```diff
@@ -3,7 +3,7 @@ class PreApplication::ProjectEnquiry::WhatProjectDoesController < ApplicationCon
   include PreApplicationContext, ObjectErrorsLogger
 
   # This method updates the what_project_does attribute of a pa_project_enquiry,
-  # redirecting to :pre_application_project_enquiry_what_will_the_project_do if successful and
+  # redirecting to :pre_application_project_enquiry_working_title if successful and
```

Comment:
```suggestion
  # redirecting to : pre_application_project_enquiry_do_you_have_a_working_title if successful and
```

Almost. We got the symbol wrong.